packages/edit-site/src/components/sync-state-with-url/use-sync-path-with-url.js

```diff
@@ -109,7 +122,7 @@ export default function useSyncPathWithURL() {
 				( navigatorLocation.path === '/wp_template' &&
 					window?.__experimentalAdminViews ) ||
```

Comment:
This part of the condition should be removed no? Now that the dataviews are not accessible from the top level.